src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -206,9 +206,17 @@ object ConsignmentFields {
   implicit val UpdateExportDataInputType: InputObjectType[UpdateExportDataInput] = deriveInputObjectType[UpdateExportDataInput]()
   implicit val StartUploadInputType: InputObjectType[StartUploadInput] = deriveInputObjectType[StartUploadInput]()
   implicit val ConnectionDefinition(_, consignmentConnections) =
-    Connection.definition[RequestContext, Connection, Consignment](
+    Connection.definition[ConsignmentApiContext, Connection, Consignment](
       name = "Consignment",
-      nodeType = ConsignmentType
+      nodeType = ConsignmentType,
+      connectionFields = fields[ConsignmentApiContext, Connection[Consignment]](
+        Field(
+          "totalPages",
+          OptionType(IntType),
+          arguments = LimitArg :: Nil,
```

Comment:
I think it's only IntelliJ that is giving a warning because when I compile from the terminal then it doesn't give any warning.
Anyway, I have moved the `consignmentConnections` after the `LimitArg` variable.